### ISSUE TITLE
Video controls become uninteractable after a certain amount of time

### ISSUE THREAD

**Issue by ttocs167** [pixeltris/TwitchAdSolutions#213]
I'm not sure what causes this, but after some time the video controls along the bottom, pause/play, volume, fullscreen etc... become unclickable. They still appear and disappear as I hover but I am unable to click them until I refresh the page.

Has anyone else had this issue?  I can't seem to find any mention of it online but it occurs for me regardless of device or browser.

**Comment by mattrxcool** [comment]
Yes I have this issue too using the Vaft js script, but I was willing to put up with it because it was successfully blocking the ads and it's not really a big deal to refresh the stream every now and again to get the controls back. But as of today it's not even blocking the ads any more :(

**Comment by seanjan10** [comment]
a partial fix but it you resize the window it will make the buttons re-interactable, which is quicker then refreshing

**Comment by xsmashx88x** [comment]
> I'm not sure what causes this, but after some time the video controls along the bottom, pause/play, volume, fullscreen etc... become unclickable. They still appear and disappear as I hover but I am unable to click them until I refresh the page.
> 
> Has anyone else had this issue? I can't seem to find any mention of it online but it occurs for me regardless of device or browser.

under ublock go to filters and add this

||supervisor.ext-twitch.tv/supervisor/v1/index.html$subdocument

**Comment by TheJellycake** [comment]
Adding this to "My Filters" worked for me

www.twitch.tv##.stream-display-ad__wrapper-visible.stream-display-ad__wrapper_lower-third.stream-display-ad__wrapper

**Comment by lazyapex** [comment]
It happened to me that I couldn't use the controls, when I uninstalled betterttv extension it didn't happen anymore

**Comment by ls612** [comment]
I only use 7tv and it happens for me too.

**Comment by pixeltris** [comment]
Should have been fixed by https://github.com/uBlockOrigin/uAssets/commit/155cb9d5d221556b04bc52dbe327e51d9726d2f7
